### PR TITLE
React Capstone Project

### PR DESCRIPTION
**Pull request by 41hulk**
## React Capstone Project: COVID-19 DATA
The main objective was to develop the home and about pages based on CC Global Summit 2015 website, designed by Cindy Shin. As we could choose the topic for this project, I came up with the idea to create a conference website about elderly care.

Additional description of the project and its features

## Home page:
- [ ] When the page loads, the web app shows the list of categories that could be filtered out by some parameter, for example by the category name.
- [ ] Along with the category name, you will display some numeric values per category that come from the API.
- [ ] When the user clicks (or taps) on a category item, the application navigates to the details page.

## Details page:
- [ ] In the details page, the web app retrieves data from the API to show detailed data in the category.
- [ ] When the user clicks on the "Back" button (<), the user navigates to the home page.

## Testing requirements:
- [ ] Unit tests for pure functions in your Redux code.
- [ ] Integration tests for your application using the React Testing Library.
- [ ] The project is a single-page application (SPA) built with React and Redux.
- [ ] The data retrieved from the API is stored in the Redux store.
- [ ] It filters the data retrieved from the API using a Filter stateless component (map).
- [ ] Every page (the main page and the pages for each item) has a unique route within the SPA.
- [ ] The project is deployed and accessible online (add a link to the online version of your app to your README file).

## Built With 🤖
- React
- Redux
- Bootstrap
- [Narrativa API](https://covid19tracking.narrativa.com/index_en.html)

## Deployed 
- [Preview](https://cvd-trak.netlify.app/) 💻

## Video Presentation 🎬
- [Description - Loom](https://www.loom.com/share/c285d18dede9414487e9ee352f646254)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+
+
+## 📝 License
+
+This project is [MIT](LICENSE) licensed.
```
Comment (elmar8287) [inline readme.md]: - [x] Please, consider that the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

- Professional PR and README 💯
- Nice commit history 👍
- Nice presentation video 👍
- Nice app design and functionality
- All tests are past 💯

## Required Changes ♻️

- [ ]  _Check the comments under the review._

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: readme.md (hunk 1 above)
Comment (41hulk) [reply readme.md]: Thanks for the comment @elmar8287

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @41hulk 👋🏻, 

Good job making the requested changes 💪🏻! Your app looks very nice 😍.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

rosycheeks-mochi-peach


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by 41hulk** [comment]
Thank you @Cathella ❤️

**Pull request merged**